src/index.js

```diff
@@ -62,6 +62,25 @@ const event = (name, data = {}) => {
   window.fbq('track', name, data)
 }
 
+/**
+ * Submit a raw query to fbq, for when the wrapper limits user on what they need.
+ * This makes it still possible to access the plain Analytics api.
+ * @param  {String} appId
+ * @param  {object} [data={}]
+ */
+const query = (...args) => {
+  if (!_fbqEnabled()) return
+
+  if (config.debug) {
```

Comment:
Maybe we should refactor `event()` to use `query()` underneath?

Reply:
Up to you! If you think that's a good pattern, then I can certainly make that change